pkg/kotsclient/channel.go

```diff
@@ -130,6 +140,80 @@ func ArchiveChannel(appID string, channelID string) error {
 	return nil
 }
 
-func GetChannel(appID string, channelID string) (interface{}, []interface{}, error) {
-	return nil, nil, nil
+var getKotsChannel = `
+query getKotsChannel($channelId: ID!) {
+  getKotsChannel(channelId: $channelId) {
+    id
+    appId
+    name
+    description
+    channelIcon
+    currentVersion
+    currentReleaseDate
+    installInstructions
+    numReleases
+    adoptionRate {
+      releaseSequence
+      semver
+      count
+      percent
+      totalOnChannel
+    }
+    customers {
+      id
+      name
+      avatar
+      actions {
+	shipApplyDocker
+      }
+      installationId
+      shipInstallStatus {
+	status
+	updatedAt
+      }
+    }
+    githubRef {
+      owner
+      repoFullName
+      branch
+      path
+    }
+    extraLintRules
+    created
+    updated
+    isDefault
+    isArchived
+    releases {
```

Comment:
do we need to fetch every release on the channel?

Reply:
@dex - see your point, but can we table that for another PR.  Currently pulling from what's on Vendorweb.